### PR TITLE
docs: update aggregate-window-functions.md

### PR DESCRIPTION
**Pull request by hanyisong**
I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Summary
this is bug?
```sql
SELECT date, AVG(amount) over (partition by date) FROM BookSold;
ERROR 1105 (HY000): Code: 1005, displayText = error:
  --> SQL:1:42
  |
1 | SELECT date, AVG(amount) over (partition by date) FROM BookSold
  | ------       ---              -          ^^ expected `)` or `,`
  | |            |                |
  | |            |                while parsing `(<expr> [, ...])`
  | |            while parsing expression
  | while parsing `SELECT ...`

.

SELECT date, AVG(amount) over () FROM BookSold;
ERROR 1105 (HY000): Code: 1005, displayText = error:
  --> SQL:1:32
  |
1 | SELECT date, AVG(amount) over () FROM BookSold
  |                                ^ expected `(`, `UNION`, `EXCEPT`, `INTERSECT`, or `SELECT`

.
```

Fixes #issue

### CODE REVIEW

**Comment by Xuanwo** [comment]
Cc @leiysky for confirmation.

**Comment by sundy-li** [comment]
Window function is not supported in new plan.  We should disable the docs for now, it's low-priority to support this. cc @soyeric128

**Comment by BohuTANG** [comment]
We can make the `window function` draft first.

**Review by soyeric128 — APPROVED** [review]

File: docs/doc/30-reference/20-functions/122-window-functions/aggregate-window-functions.md [hunk 1]
```diff
@@ -56,9 +59,13 @@ SELECT date, AVG(amount) AS avg_amount_for_branch
 FROM BookSold
 GROUP BY date;
 
-June 21|544.0
-June 22|454.5
-June 23|643.0
++---------+-----------------------+
```
Comment (BohuTANG) [inline docs/doc/30-reference/20-functions/122-window-functions/aggregate-window-functions.md:62]: The result format style is not same as others, here is a table :)

**Comment by hanyisong** [comment]
> We can make the `window function` chapter as draft first.

update window function chapter title to 'Window Functions(draft)'?

File: docs/doc/30-reference/20-functions/122-window-functions/aggregate-window-functions.md (hunk 1 above)
Comment (hanyisong) [reply docs/doc/30-reference/20-functions/122-window-functions/aggregate-window-functions.md:62]: > The result format style is not same as others, here is a table :)

Revert to previous format?

**Comment by BohuTANG** [comment]
> > We can make the `window function` chapter as draft first.
> 
> update window function chapter title to 'Window Functions(draft)'?

OK, thanks.

File: docs/doc/30-reference/20-functions/122-window-functions/aggregate-window-functions.md (hunk 1 above)
Comment (BohuTANG) [reply docs/doc/30-reference/20-functions/122-window-functions/aggregate-window-functions.md:62]: Ok

**Review by Xuanwo — APPROVED** [review]
Thanks a lot!

**Pull request merged**